src/main/java/task/Task.java

```diff
@@ -0,0 +1,63 @@
+package task;
+
+/**
+ * This class creates Task object.
+ */
+public class Task {
+
+    protected String description;
+    protected boolean isDone;
+
+    /**
+     * Constructor for creating new Task object.
+     *
+     * @param description This is the description of the Task.
+     */
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    /**
+     * This method retrieves the status icon for a Task.
+     *
+     * @return String of the icon.
+     */
+    public String getStatusIcon() {
+        return (isDone ? "\u2713" : "\u2718"); //return tick or X symbols
```

Comment:
Great that you have a comment here explaining this!! I guess you avoid putting the actual characters in due to concerns about incompatible editors that cannot show the characters?